modules/test/playwright/tests/frontend-taglib/searchPaginator.spec.ts

```diff
@@ -5,40 +5,36 @@
 
```

Comment:
if we move this file from `frontend-taglib-sample-web` to its parent folder, this means that all tests will live in the same level, no matter if they test tags from the `frontend-taglib` namespace or any other. 

I think it's fine for now as it looks simpler and we want this merged asap.

When more tags are added, we can consider other options

Reply:
As the intent is to put tests for all taglibs in `frontent-taglib` app into `frontend-taglib-sample-web`, I'd say they can be in `frontent-taglib`. I would then make internal subdivision grouping by taglib, similar to what we started doing in [portal-web](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/portal-web). In this case, it would be something like
```
/frontent-taglib/tests/liferay-ui/searchPaginator.spec.ts
```
What do you think?

Reply:
Just a quick question here, what are we going to put inside [playwright/tests/portal-web](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/portal-web) and what will go inside [playwright/tests/frontend-taglib](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/frontend-taglib) ?

The first taglibs we are adding under [playwright/tests/frontend-taglib](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/frontend-taglib) are `search-iterator` and `search-paginator` whose code is under portal-web. Regarding that I thought all our taglib tests will eventually go under frontend-taglib and will migrate stuff from portal-web (bottom and page-iterator, right now) into this module.

Am I misunderstanding this? 

I know this is a bit of a different topic than what Dani asked in this thread, because that was about the possibility of using subfolders once we have many taglibs, and this is just about whether [playwright/tests/portal-web](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/portal-web) is already a legacy folder, but if it is not then all the test code in this pull request (and the previous pull request about LPD-37458) should go under portal-web.

Reply:
I messed up a bit in https://github.com/liferay-frontend/liferay-portal/pull/4562#discussion_r1868947840, I just assumed `search-paginator` tag was in `frontend-taglib`. As it is in `portal-web`, the test in PR should be in `portal-web`

Reply:
One thing that might be causing confusion, what about `portal-web` samples that are in `frontend-taglib-sample-web`? By some logic, we could place both everything in `frontend-taglib-sample-web` PW project.

While I wouldn't oppose such an approach, I still prefer structure as in https://github.com/liferay-frontend/liferay-portal/pull/4562#discussion_r1871511263. It somehow makes a bit more sense to group by code structure, rather than sample modules structure. For example, we are already doing this with `frontend-data-set-sample-web`, that is used in both `frontend-data-set-admin-web` PW project and frontend-data-set-web PW project. We don't bundle it all in one  `frontend-data-set-sample-web` project.

Reply:
It makes sense to me tests structure perfectly matches code structure. 

We missed this in LPD-37458, because `searchPaginator.spec.ts` should have also gone under portal-web.

I'll update this pull request with these changes.

Thanks a lot!